components/combobox/combobox.jsx

```diff
@@ -307,7 +307,6 @@ const defaultProps = {
 		removePill: ', Press delete or backspace to remove',
 		selectedListboxLabel: 'Selected Options:',
 	},
-	direction: DIRECTIONS.LTR,
```

Comment:
I'm just glancing at this, but I think removal would be a breaking change to a publicly released library. Feel free to add comments and create a removal issue though.

Reply:
Makes sense, let's keep it.

Reply:
Hmm are we expected to accommodate both (1) using the `UNSAFE_Direction.Provider` context provider and (2) inputting the `direction` prop directly? It seems like if we are only supporting the first use case, we can safely remove the `direction` prop in combobox.jsx

Reply:
I would ask @davidlygagnon for the reason for this prop’s existence. 

If you all ONLY want language direction will be done by “global” Context instead of a wrapping HOC, we can remove the prop type (that is make it not a publicly documented API), but we need to keep the underlying code working and comment in the source that it should be removed in the future at a breaking change.

Reply:
We do still intend to rely on `LanguageDirection` HOC to provide the `direction` prop value.
The reason this prop exists in `Combobox` is because `Combobox` used to change its dialog alignment based on direction. This is being generalized in this PR - the dialog alignment is handled by the Dialog itself. Therefore, we don't need the prop anymore, but we can keep it around until the next breaking change.

Reply:
Yup - not needed anymore in this case.